health/health_config.c

```diff
@@ -503,6 +503,68 @@ static inline void strip_quotes(char *s) {
     }
 }
 
+static inline void alert_config_free(struct alert_config *cfg)
+{
+    freez(cfg->alarm);
+    cfg->alarm = NULL;
```

Comment:
Why do you need to set everything to NULL?

Reply:
Thanks, not needed, removed.